README.md

```diff
@@ -18,7 +18,7 @@ So, let's do it!
  Exchange | Lightning Support? | Status | Proof of Support | [Top Crypto Spot Exchanges Position](https://coinmarketcap.com/rankings/exchanges/) | Node URI | KYC on deposit or withdraw | Min. Chan Capacity (BTC)
  ------------ | ------------- | ------- | ------------- | ------------- | ------------- | ------------- | -------------
 [Binance](https://binance.com) | **YES** (JUL-2023) | :zap: | [Twitter](https://twitter.com/binance/status/1671042638592589826?s=20) | 1 | [Link](https://amboss.space/node/03a1f3afd646d77bdaf545cceaf079bab6057eae52c6319b63b5803d0989d6a72f) Binance | :question: | :question:
-[Coinbase](https://coinbase.com)| NO | :red_circle: | [Twitter](https://twitter.com/brian_armstrong/status/1644794656570617856) | 2 | n.a. | n.a. | n.a.
+[Coinbase](https://coinbase.com)| **YES**&nbsp;(APR&#8209;2024) | :zap: | [blog](https://www.coinbase.com/blog/coinbase-integrates-bitcoins-lightning-network-in-partnership-with) | 2 | n.a. | n.a. | n.a.
```

Comment:
Could you please add "💔 YES" in KYC column and " ❓" in Node URI and Min Capacity columns?